src/Sentry.Log4Net/SentryAppender.cs

```diff
@@ -79,6 +85,27 @@ protected override void Append(LoggingEvent loggingEvent)
         }
 
         var exception = loggingEvent.ExceptionObject ?? loggingEvent.MessageObject as Exception;
+
+        if (MinimumEventLevel is not null && loggingEvent.Level < MinimumEventLevel)
+        {
+            var message = !string.IsNullOrWhiteSpace(loggingEvent.RenderedMessage) ? loggingEvent.RenderedMessage : string.Empty;
+            var category = loggingEvent.LoggerName;
+            IDictionary<string, string> data = GetLoggingEventProperties(loggingEvent)
+                .Where(kvp => kvp.Value != null)
+                .ToDictionary(kvp => kvp.Key, kvp => kvp.Value!.ToString());
+
+            var level = loggingEvent.ToBreadcrumbLevel();
+            if (level is null)
+            {
+                _hub.AddBreadcrumb(message, category, type: null, data);
+            }
+            else
+            {
+                _hub.AddBreadcrumb(message, category, type: null, data, level.Value);
+            }
```

Comment:
I think this if/else block can be simplified to:
```csharp
_hub.AddBreadcrumb(message, category, type: null, data, level ?? default);
```

Reply:
Thanks! I think I've never used `default` like this before. You learn something new every day.